lms/templates/email_exists.html

```diff
@@ -8,7 +8,7 @@
     <h1 class="invalid">${_("E-mail change failed")}</h1>
     <hr class="horizontal-divider">
 
-    <p>${_("An account with the new e-mail address already exists.")}</p>
+    <p>${_("It looks like this email was already registered. Try again with a different email address.")}</p>
```

Comment:
Nit: One of my pet peeves is calling an email address an "email". But his wording *does* spell it out at the end.